packages/pipelines/src/commands/pipelines/promote.ts

```diff
@@ -201,16 +207,18 @@ export default class Promote extends Command {
       // We don't have to infer the apps or the stage they want to promote to
 
       // Strip out any empty app names due to something like a trailing comma
-      const targetAppNames = flags.to.split(',').filter(appName => appName.length >= 1)
+      const targetAppNames = flags.to.split(',').filter(appName => appName.length > 0)
 
       // Now let's make sure that we can find every target app they specified
       // The only requirement is that the app be in this pipeline. They can be at any stage.
+      // eslint-disable-next-line unicorn/no-array-reduce
```

Comment:
I see we have a few rules being added like this to individual files. Wouldn't it be more manageable to include these in the `eslintrc` file? It would also prevent future instances like this from occurring.

Reply:
Oh okay. I like the ability to find disabled rules to fix later. That makes sense and I too think that's a good approach.